### DISCUSSION TITLE
Use unique CVE for GHSA-m9hp-7r99-94h5

### DISCUSSION THREAD

**Discussion by jasinner** [dexidp/dex#1885] [General]
In order to avoid confusion when a future security advisory for Go uses CVE-2020-295[9,10,11] I recommend we issue a unique CVE for this issue. As a CNA Red Hat is able to issue a CVE for this issue as it potential affects a Red Hat product. Would you like us to issue a CVE for this issue?

**Comment by sagikazarmark** [comment]
We've been discussing this internally and it didn't seem necessary, but I don't see why we couldn't.

So I requested a CVE through GitHub.
